src/tools.c

```diff
@@ -828,20 +842,27 @@ void tools_color_chooser(const gchar *color)
 
 	if (ui_widgets.open_colorsel == NULL)
 	{
+#if (gtk_major_version == 3)	
+		ui_widgets.open_colorsel = gtk_color_chooser_dialog_new(_("Color Chooser"), GTK_WINDOW(main_widgets.window));
```

Comment:
```gtk_color_chooser_dialog_new()``` requires at least gtk version 3.4 not 3. You could use ```#if GTK_CHECK_VERSION(3, 4, 0) ... else ... endif``` for a correct version check.